src/NoFrixion.MoneyMoov/Models/Token/MerchantToken.cs

```diff
@@ -37,4 +37,33 @@ public class MerchantToken
     /// initially created. The token is not stored by NoFrixion.
     /// </summary>
     public string Token { get; set; }
+
+    /// <summary>
+    /// If set to false the merchant token will not be accepted to authorise a request.
+    /// </summary>
+    public bool IsEnabled { get; set; }
+
+    /// <summary>
+    /// Optional shared secret algorithm to use for HMAC authentication.
+    /// </summary>
+    public SharedSecretAlgorithmsEnum SharedSecretAlgorithm { get; set; }
+
+    /// <summary>
+    /// The base 64 encoded shared secret that is used for request authentication with an HMAC.
+    /// Not this property will ONLY be set when the token is initially created. It is not possible

```

Comment:
I think there is a typo here. Should the "Not" be removed?

Reply:
Thanks, it's meant to be "Note".